mlos_bench/mlos_bench/config/environments/os/linux/runtime/linux-runtime-tunables.jsonc

```diff
@@ -0,0 +1,15 @@
+{
+    "linux-kernel-runtime": {
+        "cost": 1,
+        // TODO: FIXME: These should include actual filesystem paths to the parameters.
```

Comment:
Leaving this for future work.
This PR is mostly about restructuring the configs to get them into a sane layout.